InnerEye/ML/reports/classification_crossval_report.ipynb

```diff
@@ -199,8 +199,12 @@
    "outputs": [],
    "source": [
     "if test_metrics_csv.is_file():\n",
-    "    plot_pr_and_roc_curves_from_csv(metrics_csv=test_metrics_csv, data_split=ModelExecutionMode.TEST, config=config,\n",
-    "                                    is_crossval_report=is_crossval_report)"
+    "    # When analyzing cross validation runs, the report is started with a file that contains both\n",
+    "    # validation and test set metrics in one. The test set metrics however may be missing if\n",
+    "    # inference on the test set for cross validation child runs is turned off, leading to exceptions.\n",
+    "    if not is_crossval_report or (is_crossval_report and config.inference_on_test_set):\n",
+    "        plot_pr_and_roc_curves_from_csv(metrics_csv=test_metrics_csv, data_split=ModelExecutionMode.TEST, config=config,\n",
```

Comment:
Is it cleaner to call is_inference_required here?

Reply:
no, I don't think so - entry to this code is already guarded sufficiently.